dotcom-rendering/src/types/front.ts

```diff
@@ -298,6 +298,7 @@ export type DCRFrontCard = {
 	starRating?: number;
 	webPublicationDate?: string;
 	image?: string;
```

Comment:
My suggestion as per the PR decscription would be to codify that we always expect an altText by making this `{ src: string, alt: string }`.